process_report/invoices/NERC_total_invoice.py

```diff
@@ -12,6 +12,24 @@ class NERCTotalInvoice(invoice.Invoice):
         "University of Rhode Island",
     ]
 
+    export_columns_list = [
```

Comment:
should this just be in the base invoice class? Then sublcasses can override this when needed. It looks like NERCTotalInvoice, billable_invoice, and nonbillable_invoice all export the same columns.

Reply:
@naved001  I think I prefer an empty list. I wouldn't worry about the repetition for now as it helps us highlight similarities that we can later refactor into common abstractions.

Reply:
sure, sounds good.